src/App.jsx

```diff
@@ -1,5 +1,48 @@
+import React, { useEffect, useState } from 'react'; // нужно ли импортировать в каждый файл?
+import { getToken, searchLocation } from './api';
+import CurrentWeather from './components/CurrentWeather';
+import SearchLocation from './components/SearchLocation';
+import { url, authData } from './constants';
+
 function App() {
-  return <div>Hello world!</div>;
+  const [token, setToken] = useState('');
+  const [coords, setCoords] = useState({});
+  const [searchedLocation, setSearchedLocation] = useState({});
+
+  navigator.geolocation.getCurrentPosition(
+    pos => {
+      setCoords(pos);
+    },
+    err => {
+      console.warn(`ERROR(${err.code}): ${err.message}`);
+    }
+  );
+
+  const isSearchedLocationEmpty = Object.keys(searchedLocation).length === 0;
+
+  const getSearchedLocation = location => {
```

Comment:
as i can see, your functions getSearchedLocation and getLocation don't really get anything, please rename

Reply:
I renamed them but not sure about good naming. Need to research best practicies to name callbacks which pass as props